index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width-device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Yuliya Hlukhatarenka CV</title>
+</head>
+
+<body>
+  <header>
+    <h1>Yuliya Hlukhatarenka</h1>
+    <h2> Junior Front End Developer </h2>
+  </header>
+  <aside>
+    <section class="photo-area">
+      <img src="img/foto.png" alt="photo" width="108" height="119">
+    </section>
+
+    <section class="contacts-area">
+      <h2>Contacts:</h2>
+      <p><b>Name:</b> Yuliya Hlukhatarenka</p>
+      <p><b>City:</b> Minsk, Belarus</p>
+      <p><b>Phone/cell:</b> +375295619028</p>
+      <p><b>E-mail:</b> julkin@gmail.com,</p>
+      <p>Yuliya_Hlukhatarenka@epam.com</p>
+      <p><b>Skype:</b> krisakonda</p>
+    </section>
+
+    <section class="english">
+      <h4><b>English level:</b> A2+</h4>
+    </section>
+
+    <section class="education">
+      <h2>Education</h2>
+      <p>BSU, Minsk, Belarus </p>
+      <p>Faculty of Applied Mathematics and Computer Science</p>
+      <p>Department of Discrete Mathematics and Algorithmics</p>
+      <p>Years: 1998-2003</p>
+    </section>
+  </aside>
+  <main>
+    <section class="summary-area">
+      <h3>Summary:</h3>
+      <p>I have got good practice in software development as representative of Test Department. <br>
+        I have achived good results and this became not enough for me. I was interesting in getting new
+        knowledge and improvement technical skills.
+        Deep in my soul, I always dreamed of being a developer, but I was afraid to say so.<br>
+        I have always loved to puzzle over math problems. <br>
+        Also I can add I am mindful, flexible and able to learn quickly.</p>
+    </section>
+
+    <section class="experience">
+      <h3>Experience:</h3>
+      <table>
+        <tr>
```

Comment:
You should use table tag only for presentation of tabular data, for other purposes design layouts using css